sample_deploy_config.json

```diff
@@ -5,15 +5,15 @@
   "stack_api_key_env": "IBMCLOUD_API_KEY",
   "config_order": [
     "1 - Account Infrastructure Base",
-    "2a - Security Service - Key Management",
-    "2b - WatsonX SaaS services",
+    "2 - Security Service - Key Management",
     "3a - Observability - Logging Monitoring Activity Tracker",
     "3b - Databases for Elasticsearch",
     "4 - Event Notifications",
     "5a - Security Service - Secret Manager",
     "5b - Security Service - Security Compliance Center",
-    "6 - Sample RAG app - Application Lifecycle Management",
-    "7 - Sample RAG app configuration"
+    "6 - WatsonX SaaS services",
```

Comment:
Logically the Elastic DA should be next to Watson (probably right before that). As is, Elastic sits in the middle of the Security / Observability DAs.

From the same perspective, may be we should put the Security Service DAs together, even though some of them are dependent on Event Notififcations and will not be deployed in that exact order.

Overall, I am starting to think that we might consider dropping the numbering from the member DA names - otherwise we get massive changes every time we change the order or add something in the middle. May be we should hold a vote on that.